agent/taskresource/fsxwindowsfileserver/fsxwindowsfileserver_windows.go

```diff
@@ -479,7 +478,9 @@ func (fv *FSxWindowsFileServerResource) retrieveSSMCredentials(credentialsParame
 	}
 
 	ssmClient := fv.ssmClientCreator.NewSSMClient(fv.region, iamCredentials)
-	ssmParam := filepath.Base(parsedARN.Resource)
+	// parsedARN.Resource looks like "arn:aws:ssm:us-west-2:123456789012:parameter/sample1/sample2/parameter1"
+	// We split by parameter and get ["", "/sample1/sample2/parameter1"]
+	ssmParam := strings.Split(parsedARN.Resource, "parameter")[1]
```

Comment:
`parsedARN.Resource` is guaranteed to have "parameter" in it?

Reply:
yes! if you look at the caller of this code, we use the Amazon ARN library to parse the string and make sure its a SSM parameter. This avoids any injections and other security issues.

Reply:
Thanks!